### ISSUE TITLE
Images were not downloaded

### ISSUE THREAD

**Issue by zmaniar** [iranimij/automatic-upload-images#11]
Hello. I have tried to use it but even after saving it's not saving the images and uploading it locally.

Settings: https://prnt.sc/11smvax
Product Page sample: https://uhrpalast.de/?product=approach-s10
Type of images: https://prnt.sc/11smxew

**Comment by iranimij** [comment]
Hi 
Did you have this problem on all the pages or only on this page?
I checked these images in my localhost and the plugin uploaded them successfully.

**Comment by zmaniar** [comment]
On all the pages. 

Have you checked the settings I have made?

**Comment by iranimij** [comment]
I have checked with default settings.
Did you make any changes to the settings? It seems all the settings are the same as the default.

**Comment by zmaniar** [comment]
No, I haven't made any change. I did tick and untick a few to see if it would work. 

Is there any cron that I could run for the process to get started?
